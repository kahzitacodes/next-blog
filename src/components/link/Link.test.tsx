import { render, screen } from '@testing-library/react'
import { Link } from '.'

describe('<Link />', () => {
  it('should render successfully', () => {
    const { container } = render(<Link>My link</Link>)

    expect(screen.getByText(/my link/i)).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
