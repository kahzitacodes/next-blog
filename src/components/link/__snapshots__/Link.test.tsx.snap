// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Link /> should render successfully 1`] = `
<a
  class="group text-link transition-all duration-300 ease-in-out"
>
  <span
    class="bg-gradient-to-r from-link to-link bg-[length:0%_1px] bg-left-bottom bg-no-repeat transition-all duration-400 ease-out group-hover:bg-[length:100%_1px]"
  >
    My link
  </span>
</a>
`;
